pkg/frontend/querymiddleware/results_cache_test.go

```diff
@@ -891,7 +944,8 @@ func TestFilterRecentCacheExtents(t *testing.T) {
 			},
 			shouldTruncate: []bool{false, true},
 			// First extent - 1 hour with 1000ms step is a 3601 data points - 36010 samples - not truncated
-			// Second extent - 2 hours with 1000ms step is a 7201 data points - 72010 samples - truncated to 54010 samples
+			// Second extent - 2 hours with 1000ms step is a 7201 data points - 72010 samples.
+			// Truncated to 54010 samples.
```

Comment:
nit but could be useful to say why it's truncated to 54010 exactly

Reply:
Added more details, thanks